spec/client_spec.rb

```diff
@@ -735,39 +735,6 @@ def loop_times(times)
     end
   end
 
-  describe 'campaigns', campaigns: true do
-    before(:all) do
-      @campaign_selector = SecureRandom.uuid
-      @campaign_receiver = SecureRandom.uuid
-      @campaign_sender = SecureRandom.uuid
-      @client.upsert_user({ id: @campaign_receiver, uniq: @campaign_selector, receiver: true })
-      @client.upsert_user({ id: @campaign_sender, sender: true })
-    end
-
-    it 'full flow' do
-      response = @client.create_segment({ name: 'test', type: 'user', filter: { uniq: @campaign_selector } })
-      segment = response['segment']
-      response = @client.create_campaign({ name: 'test', text: 'Hi', sender_id: @campaign_sender, segment_id: segment['id'], channel_type: 'messaging' })
```

Comment:
Let's not only remove the campaigns tests, but the actual campaign implementation as well, since it doesn't work anyways.

Reply:
Removed the implementation also